internal/storage/storageutil/object_attrs_test.go

```diff
@@ -20,6 +20,30 @@ type objectAttrsTest struct {
 
 func init() { RegisterTestSuite(&objectAttrsTest{}) }
 
+func (t objectAttrsTest) TestConvertACLRuleToObjectAccessControlMethod() {
+	projectTeam := storage.ProjectTeam{
```

Comment:
Can't we initialize the ACLRule object without creating this extra variable?

Reply:
fixed